Dependencies.props

```diff
@@ -28,4 +28,20 @@
     <Xunit_Assert_Version>2.4.1</Xunit_Assert_Version>
     <Xunit_Core_Version>2.4.1</Xunit_Core_Version>
   </PropertyGroup>
+  <PropertyGroup Label="Packages for Omex Extensions. AutoUpdate">
```

Comment:
Why is "AutoUpdate" here?
It doesn't really give the reader any context, so suggest either fleshing this out or removing it.

Reply:
It's the same marker that we are using for auto-update pipelines to identify that any updates in this group will be propagated without problems as transitive dependencies to all consumers. Also, there is a chance that we will enable package update pipelines on github also